src/models/post.ts

```diff
@@ -1,8 +1,8 @@
 import BaseModel from './bases/baseModel'
 
 export default interface Post extends BaseModel {
-	id: number
-	userid: number
-	title: string
-	body: string
+	id?: number
```

Comment:
why do we need optional here

Reply:
So that in initalState, we can have its initial value as undefined

Reply:
We dont need optional to set undefined value